src/selection.ts

```diff
@@ -1,35 +1,55 @@
 import {SingleDefChannel} from './channel';
 import {VgBinding} from './vega.schema';
 
-
 export type SelectionTypes = 'single' | 'multi' | 'interval';
 export type SelectionResolutions = 'global' | 'independent' | 'union' |
   'union_others' | 'intersect' | 'intersect_others';
 
 export interface BaseSelectionDef {
-  // domain?: SelectionDomain;
-  resolve?: SelectionResolutions;
   on?: any;
+  resolve?: SelectionResolutions;
   // predicate?: string;
-  bind?: 'scales' | VgBinding | {[key: string]: VgBinding};
+  // domain?: SelectionDomain;
```

Comment:
Okie. Maybe create an issue and add TODO link to the issue too?
(More likely that we will run into the issue than a commented code in the future.)